src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -10,6 +10,18 @@ the MIT License. See LICENSE in the project root for license information. -->
     xmlns:Controls="using:Microsoft.UI.Xaml.Controls"
     xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
     mc:Ignorable="d">
+
+    <Page.Resources>
+        <Flyout x:Key="ColorPickerFlyout">

```

Comment:
i never thought about putting a whole object into the resources

Reply:
there might be only one instance of this guy shared btwn all the objects. be careful!

Reply:
agh you're right it's shared